### ISSUE TITLE
[Word]: NVDA announcement is not proper for the equations.

### ISSUE THREAD

**Issue by WiproPSAccTest** [nvaccess/nvda#8284]
### Steps to reproduce:
1. Open the sample doc 
2. Navigate to the Equation in the document.
3. Observe the NVDA announcement for the equation.
 
### Expected behavior:
NVDA should announce all the equations as per expectation.

### Actual behavior:
NVDA is not announcing any of the character in the equation. it is just announcing the symbols and numeric value.

### System configuration:
NVDA version: 2018.1.1
App Name: Word
NVDA Installed or portable: Installed
**Other information:**
Windows version:
> Example: Windows 10 Version 1803 Build 17134.48
Name and version of other software in use when reproducing the issue:

### Other questions:
Does the issue still occur after restarting your PC?
Yes
Have you tried any other versions of NVDA?
No
> Please list them and the result

**Comment by WiproPSAccTest** [comment]
![equation_nvda](https://user-images.githubusercontent.com/22956130/40101984-ce086192-5906-11e8-99e1-66e13c830474.png)

**Comment by ehollig** [comment]
I assume this was accidentally closed.

**Comment by Adriani90** [comment]
There is no example document uploaded to this issue, we are not able to reproduce it from the description. Please open a new issue and upload an example document or comment on this one. I am closing this for now.
